tqdm/contrib/telegram.py

```diff
@@ -112,6 +122,13 @@ def clear(self, *args, **kwargs):
         if not self.disable:
             self.tgio.write("")
 
+    def close(self):
+        if self.disable:
+            return
+        super(tqdm_telegram, self).close()
+        if not (self.leave or (self.leave is None and self.pos == 0)):
```

Comment:
borrowing usual `leave=None|False|True` logic here.

Potential future issue is if people want different behaviour for terminal & telegram outputs (leave one but not the other). They'd have to override the `close()` method.

Reply:
I think the leave variable is perfect, it fits our needs. And yes in the meantime we don't worry about that case.